### DISCUSSION TITLE
SEB error 0x80070490

### DISCUSSION THREAD

**Discussion by pussurillu** [SafeExamBrowser/seb-win-refactoring#345] [Q&A]
i've already unistalled my antivirus and tryed to manually install .NET framework and VC++ but it still doesn't work. 
I cannot install the latest version because the site i'm using only supports 3.3.0.354. i don't know why it doesn't let me attach the log with all my browsers

**Comment by dbuechel** [comment]
We'd need the log of the installer to determine the exact cause. If the setup bundle fails, you should be able to install the MSI, given that you first install all required dependencies as specified in the release notes here on GitHub: https://github.com/SafeExamBrowser/seb-win-refactoring/releases/tag/3.3.0

**Comment by pussurillu** [comment]
[SEB.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/7950557/SEB.log)
i've already installed all the listed dependencies, here's the log

**Reply by dbuechel** [reply]
Yes, you're trying to install the setup bundle but this will fail as it references an outdated bootstrapper for the WebView2 runtime dependency. Thus, as already explained, if you have all the required dependencies installed you can simply install the MSI for your platform:

64-bit OS: https://github.com/SafeExamBrowser/seb-win-refactoring/releases/download/3.3.0/SEB_3.3.0.354_x64_Setup.msi
32-bit OS: https://github.com/SafeExamBrowser/seb-win-refactoring/releases/download/3.3.0/SEB_3.3.0.354_x86_Setup.msi

**Reply by pussurillu** [reply]
thank you, now it works

**Reply by bumn2024** [reply]
hi, i experienced the same thing... can u help me? here is the log
[Safe_Exam_Browser_20240423202244.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/15077550/Safe_Exam_Browser_20240423202244.log)

**Comment by bumn2024** [comment]
hi, i experienced the same thing... can u help me? here is the log
[Safe_Exam_Browser_20240423202244.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/15077550/Safe_Exam_Browser_20240423202244.log)

**Reply by dbuechel** [reply]
There appears to be a certificate issue on your system:

```
[0674:4590][2024-04-23T20:22:54]e000: Error 0x80070490: Failed to find expected public key in certificate chain.
[0674:4590][2024-04-23T20:22:54]e000: Error 0x80070490: Failed to verify expected payload against actual certificate chain.
[0674:4590][2024-04-23T20:22:54]e000: Error 0x80070490: Failed to verify signature of payload: vc_redist.x64.exe
```

Try to install the VC++ Redistributable referenced in the SEB release notes manually. Should that however fail (which is likely), then you'd need to resolve the underlying certificate issue.
